src/cljs/athens/electron/db_modal.cljs

```diff
@@ -12,11 +13,8 @@
     [athens.style :refer [color]]
     [athens.subs]
     [athens.util :refer [js-event->val]]
-    [athens.views.buttons :refer [button]]
     [athens.views.modal :refer [modal-style]]
     [athens.views.textinput :as textinput]
-    [cljsjs.react]
-    [cljsjs.react.dom]
```

Comment:
The removal of this results in this error when adding a db
![image](https://user-images.githubusercontent.com/8164667/132627766-7b1cc537-be4e-425d-ae24-415ce0f4bb8b.png)

Reply:
Fix in https://github.com/athensresearch/athens/pull/1641, thanks for the catch!